mobile/src/bundles/talent/store/actions.ts

```diff
@@ -36,7 +36,8 @@ const updateOnboardingData = createAsyncThunk<
     const { badges, hardSkills, photo, cv, ...payload } = stepPayload;
 
     if (Object.keys(payload).length === 0) {
-        return stepPayload as UserDetailsGeneralResponseDto;
+        //TODO remove unknown
+        return stepPayload as unknown as UserDetailsGeneralResponseDto;
```

Comment:
TODO ?

Reply:
@MaksymGavatuk yep, I wrote this TODO for mobile team and I told them about it